### PR TITLE
Update constraint help text

### PR DESCRIPTION
**Pull request by klaricch**

### CODE REVIEW

File: browser/help/topics/constraint.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ As mentioned above, `oe` and `LOEUF` are dependent on sample size and we note th
...
+To generate Z scores, we used a previously described, but slightly modified, sequence-context based mutational model to predict the number of expected rare (minor allele frequency < 0.1%) variants per transcript at well covered sites. We then calculated the chi-squared value for the deviation of observation from expectation for each mutational class (synonymous and missense). The square root of these values was taken and multiplied by -1 if the number of observed variants was greater than expectation or 1 if observed counts were smaller than expected. The synonymous Z scores were then corrected by dividing each score by the standard deviation of all synonymous Z scores in between -8 and 8 (-5 and 5 for gnomAD v2). For the missense Z scores, we took all Z scores between -5 and 0 (-5 and 0 for gnomAD v2) and created a mirrored distribution. The missense Z scores were then corrected by dividing each score by the standard deviation of these mirror distributions.
```
Comment (ch-kr) [inline browser/help/topics/constraint.md:65]: minor comment for this:
>For the missense Z scores, we took all Z scores between -5 and 0 (-5 and 0 for gnomAD v2)

should we reword to something like: "For the missense Z scores, we took all Z scores between -5 and 0 (same cutoffs used in gnomAD v2) ..."

**Review by ch-kr — APPROVED** [review]
one minor rewording suggestion but LGTM!

File: browser/help/topics/constraint.md (hunk 1 above)
Comment (klaricch) [reply browser/help/topics/constraint.md:65]: meant to change that to a -8

**Review by ch-kr — APPROVED** [review]
LGTM!

File: browser/help/topics/constraint.md (hunk 1 above)
Comment (ch-kr) [reply browser/help/topics/constraint.md]: oh whew I wasn't sure if we only updated LoF

**Pull request merged**